helm/boot.sh

```diff
@@ -5,13 +5,13 @@ EP_UUID="$1"; shift
 
 echo -e "\n  Preparing to start kubelet Endpoint: $EP_UUID ($EP_NAME)\n"
```

Comment:
In this file and in conftest.py below, defining $HOME/.globus_compute and referencing it is a 5 for 1, maybe more worth it?

Reply:
Good point, but as with #1067, I think it's best to constrain the scope of this PR to the re-branding effort.